code/modules/mob/living/carbon/carbon.dm

```diff
@@ -141,7 +141,9 @@
 	if(!target || !isturf(loc))
 		return FALSE
 	if(istype(target, /atom/movable/screen))
-		return FALSE
+		var/atom/movable/screen/S = target
+		if(!S.can_throw_target)
```

Comment:
how does a screen atom have the can_throw_target var be relevant

Reply:
This is apart of a complex solution to the right click denial technology. Do you want me to walk you through it?